lib/IRGen/GenEnum.cpp

```diff
@@ -5794,7 +5794,9 @@ std::unique_ptr<EnumImplStrategy>
 EnumImplStrategy::get(TypeConverter &TC, SILType type, EnumDecl *theEnum) {
   unsigned numElements = 0;
   TypeInfoKind tik = Loadable;
-  IsFixedSize_t alwaysFixedSize = IsFixedSize;
+  IsFixedSize_t alwaysFixedSize =
```

Comment:
I would prefer leaving the old line in place and adding a separate 'if' statement to set it to IsNotFixedSize, but that's up to you.

Thanks for figuring out the right fix! Looks like this has been broken ever since I added this code in 2015 🤦‍♂️